cranelift/codegen/src/loop_analysis.rs

```diff
@@ -190,6 +190,19 @@ impl LoopAnalysis {
         self.valid = false;
     }
 
+    // Determines if a block dominates any predecessor
+    // and thus is a loop header
+    fn check_loop_header(
```

Comment:
Let's call this `is_loop_header` (`check` doesn't tell me anything about what it returns; usually implies an invariant check or something like that instead). And the doc-comment should end with a period.

Reply:
Was going to use `is_loop_header` but already in use by another function - check was best synonym but not ideal